packages/wdio-spec-reporter/src/index.ts

```diff
@@ -69,7 +70,16 @@ export default class SpecReporter extends WDIOReporter {
                 return this._originalStdoutWrite(chunk, encoding, callback)
             }
         }
+        this._chalk = new Chalk(options.color === false ? { level:0 } : {})
+    }
 
+    /**
+     * @param color color in which you want the message to appear
+     * @param msg the message to print in terminal
+     * @returns colord value based on chalk to print in terminal
+     */
+    setMessageColor(color:'green' | 'cyan' | 'red' | 'gray' | 'grey', message:string): string{
```

Comment:
I don't think we need this method as it barely abstracts logic, let's keep the implementation as it was e.g. `${chalk[this.getColor(state)](this.getSymbol(state))} ${title}`

Reply:
If i remove this function, then i cannot write proper testcases. 

As in tests, how will i check if my color attribute works properly or not? 

That's the sole reason i have written this function and it abstracts this thing:

```chalk[function()](function())```  <--- this looks bad. isn't it? 

So, why not abstract this thing rather than writing it everywhere?
It is actually a good thing, i believe.

Reply:
ok, one request then, can we include the `this.getColor(state)` within the `setMessageColor` call then?
so it becomes `setMessageColor(state, message)`

Reply:
enums added and snapshot updated
  - ChalkColor and State enum added
  - snapshot updated
  - code is refactored accordingly

Reply:
@christian-bromann should i standardize the usage of chalk.bold throughout the project?
Which one would be more appropriate `chalk.bold.cyan('RUNNING')` or `chalk.bold(chalk.yellow("some text"))`?

Reply:
I am converting code according this approach `chalk.bold(chalk.yellow("some text"))`.

Reply:
Makes sense! It allows us to verify that `bold` and `yellow` was called. 👍